src/locales/en/translations.json

```diff
@@ -1,7 +1,5 @@
 {
```

Comment:
Again with my weird question. I thought we dont need those (?)

Reply:
Not weird and you're right. See: [here](https://github.com/dhis2/capture-app/pull/751#discussion_r471363149)